openapi-spec/swagger.yaml

```diff
@@ -0,0 +1,1467 @@
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '409':
+          description: An item already exists
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '500':
+          description: An unexpected error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+  '/v1/storages/{storage_id}':
+    get:
+      tags:
+        - Storages
+      description: Get details of a storage device
+      operationId: GetStorageBackendbyID
+      parameters:
+        - name: storage_id
+          in: path
+          description: Database ID created for a storage backend .
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Storage backend list available
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StorageBackendResponse'
+        '401':
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '500':
+          description: An unexpected error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+    delete:
+      tags:
+        - Storages
+      description: Unregister an already registered storage backend
+      parameters:
+        - name: storage_id
+          in: path
+          description: Database ID created for a storage backend .
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+      responses:
+        '202':
+          description: Accepted
+        '401':
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '500':
+          description: An unexpected error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+  /v1/storages/sync-all:
+    post:
```

Comment:
suggest use /v1/storage/sync to replace sync-all

Reply:
done